src/shadowbox/server/outline_shadowsocks_server.ts

```diff
@@ -82,7 +83,8 @@ export class OutlineShadowsocksServer implements ShadowsocksServer {
     if (this.verbose) {
       commandArguments.push('-verbose');
     }
-    this.ssProcess = child_process.spawn('/root/shadowbox/bin/outline-ss-server', commandArguments);
+    const sbBinDir = process.env.SB_BIN_DIR || '/root/shadowbox/bin';
```

Comment:
Same, let's pass the binary location in the constructor.

Reply:
Done